submissions/semenjob/script/index.js

```diff
@@ -0,0 +1,52 @@
+window.addEventListener('DOMContentLoaded', () => {
+  initMusicApp()
+})
+
+function initMusicApp() {
+  const tabs = document.querySelectorAll('.wrapper__item'), contentTitle = document.querySelector('.music__title'), contentDescr = document.querySelector('.music__descr'), parentItems = document.querySelector('.wrapper__list')
+
```

Comment:
Please, use Prettier :) https://prettier.io/playground/

It will make code easier to read

Reply:
ok